### PR TITLE
Update Innity Adapter to Prebid.js v1.0

### PR DESCRIPTION
**Pull request by engtat**
## Type of change
- [x] New bidder adapter

## Description of change
Migrate to Prebid.js v1.0

- test parameters for validating bids
```
{
  bidder: 'innity',
  params: {
    pub: 267,
    zone: 62546
  }
}
```

- [x] official adapter submission

## Docs repo PR
https://github.com/prebid/prebid.github.io/pull/630

### CODE REVIEW

File: modules/innityBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  },
+  buildRequests: function(validBidRequests) {
+    return validBidRequests.map(bidRequest => {
+      let parseSized = utils.parseSizesInput(bidRequest.sizes);
+      let arrSize = parseSized[0].split('x');
+      return {
+        method: 'GET',
+        url: ENDPOINT,
+        data: {
+          cb: new Date().getTime(),
```
Comment (jaiminpanchal27) [inline modules/innityBidAdapter.js]: It is recommended to use `utils.timestamp()`

File: modules/innityBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      height: res.height,
+      creativeId: res.creative_id,
+      currency: 'USD',
+      netRevenue: true,
+      ttl: 60,
+      ad: '<script src="' + location.protocol + '//cdn.innity.net/frame_util.js"></script>' + res.tag,
+    };
+    return [bidResponse];
+  },
+  getUserSyncs: function(syncOptions, serverResponses) {
```
Comment (jaiminpanchal27) [inline modules/innityBidAdapter.js]: You can remove this function if you are not doing userSyncs.

**Comment by jaiminpanchal27** [comment]
Also please submit a PR to the [docs repo](https://github.com/prebid/prebid.github.io) to add a file for your adapter to the [bidders directory](https://github.com/prebid/prebid.github.io/tree/master/dev-docs/bidders) so your adapter's params will appear on the [bidders page](http://prebid.org/dev-docs/bidders.html). Thank you for contributing

**Comment by engtat** [comment]
Changes updated. For PR to the doc repo, I actually posted this earlier in my PR (https://github.com/prebid/prebid.github.io/pull/630) or I am missing something? Thanks.

**Pull request merged**